parse/grammar/KuneiformParser.g4

```diff
@@ -279,34 +279,41 @@ delete_statement:
     (WHERE where=sql_expr)?
 ;
 
+// https://docs.kwil.com/docs/kuneiform/operators
 sql_expr:
-    literal type_cast?                                                              # literal_sql_expr
+    // highest precedence:
```

Comment:
I suggest we put expr with same shape next to each other, different shape expr has no precedence issues

Reply:
It's effectively the same thing